app/controllers/mail-moderations/reject.js

```diff
@@ -1,7 +1,7 @@
 import { inject as service } from '@ember/service';
 import Controller from '@ember/controller';
 import { isInvalidResponse } from 'ember-fetch/errors';
-
+// todo: octane
```

Comment:
Did you forget something here or is this for a later PR? This comment is in some other files too.

Reply:
for later PR